main.go

```diff
@@ -183,7 +183,12 @@ func visit(url *url.URL) {
 		log.Fatalf("unable to create request: %v", err)
 	}
 	for _, h := range httpHeaders {
-		req.Header.Add(headerKeyValue(h))
+		k, v := headerKeyValue(h)
+		if strings.EqualFold(k, "host") {
```

Comment:
Do we want `host` or `Host`? We should probably use this function to ensure they are following the same format rules:

* https://golang.org/pkg/net/http/#CanonicalHeaderKey